### DISCUSSION TITLE
Issues saving scenario configuration

### DISCUSSION THREAD

**Discussion by ruben-bc** [InterDigitalInc/AdvantEDGE#349] [Q&A] [answered]
Hi everyone,

When I'm creating a new scenario or saving one previously imported, sometimes it doesn't save properly and then it's impossible to open that scenario again (because it's not in the select menu), and even when editing the map or configuration it doesn't saves. What happens is that the **delete** button in the configure pane turns gray, instead of normal blue, and since this moment is impossible to save anything... no matter how many times you click the button **save**. I want to test some things, but without saving the whole scenario configuration I can't run things right.

It's not the first time this happens to me, but now I'm doing a new scenario and suddenly I have this problem continuously. It's so weird and I don't understand how it can be happening. Any idea?

Thanks so much in advance!

**Comment by roymx** [comment, marked as answer]
Hello @rubenbc28 ,

There may be several reasons why a scenario does not get saved (error in scenario validation, storage unavailable, etc.)
What version of the platform are you using?
Is the problem happening with any scenarios or just that one?
Does it work when you create a basic scenario from scratch and save it?

First debugging steps should be...
1- Are all pods running... especially meep-platform-ctrl and couchdb (`kubectl get pods --all-namespaces`) and is the indicator green in the upper right corner of the browser.
2- Look at the logs from meep-platform-ctrl when you perform the failing save operation (when you hit OK) to make sure the platform receives it 
3- In the browser (assuming chrome) hit F12 and look at the REST operation in the Network tab when you hit OK.
You should see a `200` on a `PUT/POST https://<your-platform-address>/platform-ctrl/v1/scenarios/<scenario-name>` if you don't get 200 what are you getting & what's the error message?

Also, if you can share the scenario we can try load/save it over here and tell you what we see.

**Comment by ruben-bc** [comment]
Hello Mike,

After running the second debug option you told me, I saw in the log an error with the configuration of one element that was the reason of the failing. Maybe another time happened me the same and I didn't realize without seeing the record of these pod, because I have the latest version of the platform and it ocurred with both, imported scenario and new from scratch. And of course the returning message in network option it's ok, so there isn't any additional problem.

Thank you so much for the help!
